src/Storefront/Resources/app/storefront/src/plugin/cookie/cookie-configuration.plugin.js

```diff
@@ -70,13 +71,17 @@ export default class CookieConfiguration extends Plugin {
         this._httpClient = new HttpClient();
 
         this._registerEvents();
+        this._checkCookieConfigurationHash();
 
         document.$emitter.subscribe('CookieConfiguration/requestConsent', (payload) => {
             if (payload instanceof CustomEvent) {
                 payload = payload.detail;
             }
             this.openRequestConsentOffCanvas(payload.route, payload.cookieName);
         });
+
+        // Subscribe to offcanvas close event to show cookie bar again when user closes without making a choice
```

Comment:
Just for my understanding. It will show the cookie BAR again (at the bottom) and not the OffCanvas?

Reply:
Yes, when you close the OffCanvas with the X but do not accept (at least) the technical requirements, the cookie bar at the bottom will appear again.